src/ctelnet.cpp

```diff
@@ -305,7 +305,7 @@ void cTelnet::disconnect()
 
 void cTelnet::handle_socket_signal_error()
 {
-    QString err = "[ ERROR ] - TCP/IP socket ERROR:" % socket.errorString();
+    QString err = tr("[ ERROR ] - TCP/IP socket ERROR:") % socket.errorString();
```

Comment:
Whilst this *is* a valid use of the `QStringBuilder::operator%` it mandates that the error message is tacked onto the end of the message -  it would have been more translatable to use:
```
     QString err = tr("[ ERROR ] - TCP/IP socket ERROR: \"%1\".").arg(socket.errorString());
```
Notice that there is now a space after the colon which is missing in the original (this is a common feature in the original texts in this PR isn't it! :stuck_out_tongue_closed_eyes:) as well as a terminal `.`. It also allows the `socket.errorString()` to be rearranged (as `%1`) within the body of the translated text should that be necessary/desirable for a particular locale/language.